test/snippet/search/configuration_default.cpp

```diff
@@ -2,15 +2,12 @@
 
 int main()
 {
-    using namespace seqan3;
-//![default]
-// No errors, all hits as text position
-configuration const default_cfg = search_cfg::max_error{search_cfg::total{0},
-                                                        search_cfg::substitution{0},
-                                                        search_cfg::insertion{0},
-                                                        search_cfg::deletion{0}}
-                                  | search_cfg::output{search_cfg::text_position}
-                                  | search_cfg::mode{search_cfg::all};
-//![default]
+    // No errors, all hits as text position
+    seqan3::configuration const default_cfg = seqan3::search_cfg::max_error{seqan3::search_cfg::total{0},
+                                                                            seqan3::search_cfg::substitution{0},
+                                                                            seqan3::search_cfg::insertion{0},
+                                                                            seqan3::search_cfg::deletion{0}}
+                                            | seqan3::search_cfg::output{seqan3::search_cfg::text_position}
+                                            | seqan3::search_cfg::mode{seqan3::search_cfg::all};
```

Comment:
for everything that isn't a view, the `|` goes on the end of line I think... @smehringer do you recall the decision?

Reply:
https://github.com/seqan/seqan3/wiki/General-formatting says at the end if it's not a range, otherwise (context=range) align with previous `|` or `=`

Reply:
So should I keep this or change it?

Reply:
Move `|` to the end!